src/instrument/devices/bd_5a.py

```diff
@@ -0,0 +1,45 @@
+"""
+BD-5a Device in station 8-ID-D
+"""
+
+__all__ = """
+    bd5a
+""".split()
+
+
+import logging
+
+from ophyd import Device
+from ophyd import EpicsMotor
+from ophyd import FormattedComponent as FCpt
+
+logger = logging.getLogger(__name__)
+logger.info(__file__)
+
+
+class BD5A(Device):
```

Comment:
Could this be simpler?

```py
class BD5A(Device):
    h = Component(EpicsMotor, "m9", labels={"motors"})
    v = Component(EpicsMotor, "m10", labels={"motors"})

bd5a = BD5A("8iddSoft:CR8-D1:US:", name="bd-5a")
```

Reply:
I'm thinking that `BD5A` is not intended for any re-use.  Is that right?

Reply:
Possibly, but I like the idea of the format being exactly the same as in all the other places that have a device with multiple motors